README.md

```diff
@@ -5,8 +5,16 @@ formatted strings (OS X only).
 
 # Installation
 
+To install the stable release:
+
     pip install pagebotcocoa
 
+To setup the development environment and install the version from this git repo:
+
+    virtualenv venv -ppython3
```

Comment:
you can just use the built-in `venv` module instead of `virtualenv`, e.g. `python3 -m venv YOUR_ENV_NAME`